tests/foreman/cli/test_setting.py

```diff
@@ -86,3 +86,129 @@ def test_negative_discover_host_with_invalid_prefix(self):
 
         :caseautomation: notautomated
         """
+
+    @stubbed
+    @tier1
+    def test_positive_update_email_delivery_method_smtp(self):
+        """Check Updating SMTP params through settings subcommand
+
+        :id: b26798e9-528c-4ed6-a09b-dc8e4668a8d7
+
+        :steps:
+            1. set "delivery_method" to smtp
```

Comment:
ah.. Ignore, both smtp and sendmail are supported and you covered that in below test. thanks